spring-cloud-dataflow-rest-client/src/test/java/org/springframework/cloud/dataflow/rest/client/DataflowTemplateTests.java

```diff
@@ -111,16 +111,15 @@ public void testThatObjectMapperGetsPrepared() {
 
 	@Test
 	public void testJobParameters() throws JsonProcessingException {
-		final ObjectMapper objectMapper = new ObjectMapper();
 		JobParametersBuilder jobParametersBuilder = new JobParametersBuilder();
 		jobParametersBuilder.addString("foo", "foo");
 		jobParametersBuilder.addString("bar", "bar");
 
 		JobParameters jobParameters = jobParametersBuilder.toJobParameters();
-		DataFlowTemplate.prepareObjectMapper(objectMapper);
-		assertCorrectMixins(objectMapper);
-		String jobParametersSerialized = objectMapper.writeValueAsString(jobParameters);
-		jobParameters = objectMapper.readValue(jobParametersSerialized, JobParameters.class);
+		DataFlowTemplate.prepareObjectMapper(this.mapper);
```

Comment:
I believe prepareObjectMapper does the exact things to `this.mapper` that is done in `@BeforeEach`. Maybe `this.mapper` in @BeforeEach` could use `prepareObjectMapper`? I could me missing something though.